main.tex

```diff
@@ -531,27 +504,31 @@ \subsection*{Cross-resistance between pyrethroids and DDT}
 %
 The \texttt{L995F} and \texttt{L995S} alleles are known to increase resistance to both of these insecticide classes \cite{Martinez-Torres1998,Ranson2000}.
 %
-Except in a few locations, DDT has not been used for IRS within the last two decades, and is not suitable for use in bed-nets due to potential carcinogenicity in humans \cite{Turusov2002}.
+Except in a few locations, DDT has not been used for IRS within the last two decades, and is not suitable for use in bed-nets due to potential carcinogenic effects \cite{Turusov2002}.
 %
 DDT was, however, used in Africa for several pilot IRS projects carried out during the first global campaign to eradicate malaria, during the 1950s and 1960s \cite{Davies2007b}.
 %
 DDT was also used in agriculture from the 1940s, and although agricultural usage has greatly diminished since the 1970s, some usage may remain \cite{Dunlap1981, Turusov2002}.
 %
-In this study we reported evidence of positive selection on haplotypes carrying \texttt{L995F} and haplotypes carrying \texttt{L995S}, as well as haplotypes carrying the \texttt{I1527T+V402L} combination.
+In this study we reported evidence of positive selection on the \texttt{L995F} and \texttt{L995S} alleles, as well as the \texttt{I1527T+V402L} combination and possibly also \texttt{M490I}.
 %
 We also found 14 other non-synonymous substitutions within \textit{Vgsc} that have arisen in association with \texttt{L995F} and appear to be positively selected.
 %
 Given that pyrethroids have dominated public health insecticide use for two decades, it is reasonable to assume that the selection pressure on these alleles is primarily due to pyrethroids.
 %
-If a resistance allele had been selected primarily by the earlier phase of DDT usage, then we might expect to see a faster rate of EHH decay for haplotypes carrying that allele relative to other alleles selected by recent pyrethroid usage.
+The \texttt{L995S} allele has a stronger DDT resistance phenotype than \texttt{L995F}, and it has been suggested that \texttt{L995S} may have been primarily selected by DDT usage @@REF.
+%
+However, we did not find any systematic difference in the extent of haplotype homozygosity between these two alleles, suggesting that both alleles have been under selection over a similar time frame.
```

Comment:
Think we need to explain better here that we are comparing L995F versus L995S. Is there a reference we can cite regarding L995S being selected by DDT use?

Reply:
No, I don't think so. IIRC this is part of the accepted dogma, but when I tried to follow it up there actually was experimental evidence.

Reply:
There is evidence that 995S is less effective against pyrethroids than 995F, and the reverse true for DDT (Ranson 2000) but I don't think that is the same as what you are asking.